build/archive/contents/README

```diff
@@ -19,7 +19,7 @@ To interact with a one-node SQL cluster:
 
      $ cockroach start --background --insecure
      $ cockroach sql --insecure
-     $ cockroach quit --insecure
+     $ killall -TERM cockroach # to shut down the nodes
```

Comment:
One thing to note here is that on some Unices (looking at you Solaris), `killall` will kill _all_ processes on the system. Maybe that's obscure enough that we don't have to worry about it, given that we only officially support Linux and macOS does something similar here. But worth keeping in mind if we're going to sprinkle this all over our docs etc.

Reply:
Good point, Erik! Personally, I never run `killall`. In this case, I'd probably use `pgrep cockroach | xargs kill -TERM`, but I am also fine with `killall` being in README.